3.distributed-calculator/README.md

```diff
@@ -9,7 +9,90 @@ This sample shows method invocation and state persistent capabilities of Dapr th
 
 The front-end application consists of a server and a client written in [React](https://reactjs.org/). 
 Kudos to [ahfarmer](https://github.com/ahfarmer) whose [React calculator](https://github.com/ahfarmer/calculator) 
-sample was used for the client. The following architecture diagram illustrates the components that make up this sample: 
+sample was used for the client.
+
+## Running locally
+
+We will be starting the four operator apps (add, subtract, multiply and divide) along with the dapr sidecar locally and then run the front end app which persists the state in a local redis state store.
```

Comment:
Please do not use "we" in samples or docs. Samples should follow similar guidelines to docs. Please see:  https://github.com/dapr/docs/blob/master/contributing/README.md

Reply:
Fixed